python/tvm/contrib/hexagon/session.py

```diff
@@ -286,7 +286,9 @@ def get_graph_debug_executor(
             graph_json, graph_debug_mod, self.device, dump_root=str(dump_root)
         )
 
-    def get_executor_from_factory(self, module: Union[ExecutorFactoryModule, relax.Executable]):
+    def get_executor_from_factory(
+        self, module: Union[ExecutorFactoryModule, relax.Executable], hexagon_arch: str = "v68"
+    ):
         """Create a local GraphModule which consumes a remote libmod.
 
         Parameters
```

Comment:
I think we need to add the docstring for `hexagon_arch`.

Reply:
Oh right, thanks for the catch.

Reply:
I've added the docstring